Solutions/DNS Essentials/Analytic Rules/ExcessiveNXDOMAINDNSQueriesAnomalyBased.yaml

```diff
@@ -118,8 +118,10 @@ eventGroupingSettings:
 customDetails:
   DNSQueries: DNSQueries
   AnomalyScore: score
+  baseline: baseline
+  Total: Total
 alertDetailsOverride:
-  alertDisplayNameFormat: "[Anomaly] Excessive NXDOMAIN DNS Queries has been detected from ClientIP: '{{SrcIpAddr}}'"
-  alertDescriptionFormat: "Client source is generating excessive amount of DNS queries for non-existent domains.\n\nBaseline for 'NXDOMAIN' error count for this client: '{{baseline}}'\n\nCurrent 'NXDOMAIN' error count for this client: '{{Total}}'"
+  alertDisplayNameFormat: "[Anomaly] Excessive NXDOMAIN DNS Queries has been detected from Client IP: '{{SrcIpAddr}}'"
```

Comment:
probably "client" and not "Client". Commented here, but the capitalization in mid-sentence note applies in many places.

Reply:
thanks. done